lib/components/Pane/Body.js

```diff
@@ -18,16 +18,16 @@ const Body = ({ children, className, hasFooter = true }) => (
 
 Body.propTypes = {
   /**
-   * To specify if the pane has a footer.
+   * To specify if the Pane has a Footer.
```

Comment:
I think here we should use "footer" instead of "Footer" because we don’t have a component Footer. The component we have is Pane.Footer.

@karthiknmenon Could you please share your thoughts

Reply:
Yes. here it should be "footer" as we are not referring it to a component. @deepakjosp